rpi-eeprom-config

```diff
@@ -98,9 +98,9 @@ class BootloaderImage(object):
 
 def main():
     parser = argparse.ArgumentParser('RPI EEPROM config tool')
-    parser.add_argument('--config', help='Filename of new bootloader config')
-    parser.add_argument('--out', help='Filename for the EEPROM image with updated config')
-    parser.add_argument('eeprom', help='EEPROM filename (pieeprom.bin)')
+    parser.add_argument('--config', help='Insert new bootloader config into EEPROM image. CONFIG specifies the file containing the new config.')
+    parser.add_argument('--out', help='Extract current bootloader config from EEPROM image. OUT specifies the file to put the config into.')
```

Comment:
Changes are an improvement but I've spotted that '--out' is not quite correct. It's just the name of the output file which can be the EEPROM if run as 
rpi-eeprom-config --config new-config.txt --out new-eeprom.bin current-eeprom.bin

Reply:
Ah, OK. So --out is used for two options - when run with just --out as a switch, it dumps the config, and when run in combination with --config it instead copies an existing EEPROM image, rewriting the config as specified by CONFIG in the process - is that correct?

Reply:
Yes, '--out' specifies a filename for the output, if --out is omitted then the output is written to stdout so

These commands are equivalent
rpi-eeprom-config pieeprom.bin > boot.conf 
rpi-eeprom-config pieeprom.bin --out boot.conf 

and so are these
rpi-eeprom-config pieeprom.bin --config boot.conf > new-eeprom.bin
rpi-eeprom-config pieeprom.bin --config boot.conf --out new-eeprom.bin

Perhaps it would have been better if I hadn't added --out and called --config something like --update-config. Oh well :)

Reply:
It makes sense to have an option to output to a file, since noobs don't know you can pipe output to a file.

I will amend my PR.